dts/bindings/spi/nordic,nrf-spi-common.yaml

```diff
@@ -15,14 +15,21 @@ properties:
     sck-pin:
       type: int
       required: true
-      description: SCK pin
+      description: |
+        SCK pin number. The value contains a GPIO pin in bits 0:4,
+        and the port number in bit 5.
+
+        For the port number, use 0 for P0, and 1 for P1. The P1 bit is
```

Comment:
Isn't it obvious?

Reply:
It may be, but isn't it better to be explicit?